### PR TITLE
Unpopulus

### PR DESCRIPTION
**Pull request by konradkonrad**
This tries to remove `populus` and `ethereum` (1.6.1) `tester`.

### CODE REVIEW

File: microraiden/test/fixtures/accounts.py [hunk 1]
```diff
@@ -43,7 +41,7 @@ def private_keys(private_key_seed: int) -> List[str]:
     Note: using 0 as the seed computes tester coinbase as the first address. This might cause
     issues, especially on sweeping all of these accounts, as the coinbase cannot be swept.
     """
-    return [encode_hex(keccak(str(private_key_seed + i))) for i in range(10)]
+    return [encode_hex(keccak(text=str(private_key_seed + i))) for i in range(10)]
```
Comment (konradkonrad) [inline microraiden/test/fixtures/accounts.py]: this one is unproven yet

File: microraiden/test/fixtures/accounts.py [hunk 2]
```diff
@@ -56,16 +54,16 @@ def faucet_private_key(request, faucet_password_path: str) -> str:
     private_key = request.config.getoption('faucet_private_key')
     if is_hex(private_key):
         assert len(remove_0x_prefix(private_key)) == 64
-        return private_key
+        return bytes(private_key, 'ascii')
```
Comment (konradkonrad) [inline microraiden/test/fixtures/accounts.py]: encoding is unproven

File: microraiden/test/fixtures/accounts.py [hunk 3]
```diff
@@ -56,16 +54,16 @@ def faucet_private_key(request, faucet_password_path: str) -> str:
...
     else:
         private_key = get_private_key(private_key, faucet_password_path)
         assert private_key is not None, 'Error loading faucet private key from file.'
-        return private_key
+        return bytes(private_key, 'ascii')
```
Comment (konradkonrad) [inline microraiden/test/fixtures/accounts.py]: encoding is unproven

File: microraiden/test/fixtures/accounts.py [hunk 4]
```diff
@@ -56,16 +54,16 @@ def faucet_private_key(request, faucet_password_path: str) -> str:
...
 
 
 @pytest.fixture(scope='session')
 def faucet_address(faucet_private_key: str):
-    return privkey_to_addr(faucet_private_key)
+    return privkey_to_addr(faucet_private_key.decode('ascii'))
```
Comment (konradkonrad) [inline microraiden/test/fixtures/accounts.py]: encoding is unproven

File: microraiden/test/fixtures/accounts.py [hunk 5]
```diff
@@ -81,23 +79,21 @@ def make_account(
...
+            web3.personal.importRawKey(private_key, 'nopassphrase')
         fund_account(
             address,
             eth_allowance,
             token_allowance,
             token_contract,
             web3,
             wait_for_transaction,
-            faucet_private_key
+            faucet_private_key.decode('ascii')
```
Comment (konradkonrad) [inline microraiden/test/fixtures/accounts.py]: encoding is unproven

**Comment by pcppcp** [comment]
@konradkonrad take a look at `web3` fixture in `raiden-libs`. It works without populus, with up-to-date eth-tester and is based on microraiden fixtures code.

**Comment by konradkonrad** [comment]
deprecated.

**Pull request closed without merging**